### PR TITLE
[Commands] Fix force unwrap

### PR DESCRIPTION
**Pull request by unnamedd**
to get the currentWorkingDirectory

### CODE REVIEW

File: Sources/Commands/SwiftTool.swift [hunk 1]
```diff
@@ -766,7 +766,8 @@ private func getEnvBuildPath() -> AbsolutePath? {
     // Don't rely on build path from env for SwiftPM's own tests.
     guard POSIX.getenv("IS_SWIFTPM_TEST") == nil else { return nil }
     guard let env = POSIX.getenv("SWIFTPM_BUILD_DIR") else { return nil }
-    return AbsolutePath(env, relativeTo: localFileSystem.currentWorkingDirectory!)
+    guard let currentWorkingDirectory = localFileSystem.currentWorkingDirectory else { return nil }
```
Comment (aciidb0mb3r) [inline Sources/Commands/SwiftTool.swift]: We should move `currentWorkingDirectory` to be a parameter to `getEnvBuildPath()` method.

Comment (unnamedd) [reply Sources/Commands/SwiftTool.swift]: @aciidb0mb3r do you think okay this parameter naming?

File: Sources/Commands/SwiftTool.swift [hunk 2]
```diff
@@ -762,11 +764,14 @@ private func findPackageRoot() -> AbsolutePath? {
     return root
 }
 
-private func getEnvBuildPath() -> AbsolutePath? {
+/// Returns path of environment build path relative to the directory
+private func getEnvBuildPath(relativeTo buildPath: AbsolutePath?) -> AbsolutePath? {
```
Comment (aciidb0mb3r) [inline Sources/Commands/SwiftTool.swift]: This doesn't need to be optional and we can just call it workingDir.

```
private func getEnvBuildPath(workingDir: AbsolutePath?) -> AbsolutePath?
```

Comment (unnamedd) [reply Sources/Commands/SwiftTool.swift]: Done! 😁

**Comment by aciidb0mb3r** [comment]
@swift-ci smoke test

File: Sources/Commands/SwiftTool.swift (hunk 2 above)
Comment (rrecio) [reply Sources/Commands/SwiftTool.swift]: @aciidb0mb3r just so I can understand better your thinking process, why workingDir instead of buildPath?

**Comment by unnamedd** [comment]
@aciidb0mb3r tests have failed :( for some reason the problem doesn't look to be in the code:

```shell
16:47:10 env: '/home/buildnode/jenkins/workspace/swift-package-manager-Linux-smoke-test/build/Ninja-ReleaseAssert/swiftpm-linux-x86_64/x86_64-unknown-linux/release/swift-test': No such file or directory
```

Do I have to do something to help with this problem?

File: Sources/Commands/SwiftTool.swift [hunk 3]
```diff
@@ -398,9 +398,11 @@ public class SwiftTool<Options: ToolOptions> {
         let packageRoot = findPackageRoot()
 
         self.packageRoot = packageRoot
-        self.buildPath = getEnvBuildPath() ??
+        
+        let buildPath = getEnvBuildPath(workingDir: cwd) ??
             customBuildPath ??
-            (packageRoot ?? localFileSystem.currentWorkingDirectory!).appending(component: ".build")
+            (packageRoot ?? cwd)
+        self.buildPath = buildPath.appending(component: ".build")
```
Comment (aciidb0mb3r) [inline Sources/Commands/SwiftTool.swift]: The `.build` should only be appended in the last case, i.e. it should be `... (packageRoot ?? cwd).appending(component: ".build")`

**Comment by aciidb0mb3r** [comment]
The failure looks related to your change, I suggested a possible solution.

File: Sources/Commands/SwiftTool.swift (hunk 3 above)
Comment (unnamedd) [reply Sources/Commands/SwiftTool.swift]: Make sense, thank you @aciidb0mb3r! :)

**Pull request merged**